src/main/scala/no/ndla/frontpageapi/model/api/UpdatedMetaDescription.scala

```diff
@@ -0,0 +1,3 @@
+package no.ndla.frontpageapi.model.api
+
+case class UpdatedMetaDescription(metaDescription: String, language: String)
```

Comment:
Denne kan vel extende LanguageField.

Reply:
Tror egentlig det var feil at den api.UpdatedAboutSubject extender LanguageField, det trengs vel bare for domain?

Reply:
Ja det er sant.